automated install/basic-install.sh

```diff
@@ -1001,6 +1001,36 @@ remove_old_dnsmasq_ftl_configs() {
     fi
 }
 
+remove_old_pihole_lighttpd_configs() {
+    local lighttpdConfig="/etc/lighttpd/lighttpd.conf"
+    local condfd="/etc/lighttpd/conf.d/pihole-admin.conf"
+    local confavailable="/etc/lighttpd/conf-available/15-pihole-admin.conf"
+    local confenabled="/etc/lighttpd/conf-enabled/15-pihole-admin.conf"
+
+
+    if [[ -f "${lighttpdConfig}" ]]; then
```

Comment:
You could also remove this outer `if` here and only check for `if grep -q -F 'include "/etc/lighttpd/conf.d/pihole-admin.conf"' "${lighttpdConfig}"; then`